src/js.cookie.js

```diff
@@ -67,7 +67,7 @@
 					attributes.expires && '; expires=' + attributes.expires.toUTCString(), // use expires attribute, max-age is not supported by IE
 					attributes.path    && '; path=' + attributes.path,
 					attributes.domain  && '; domain=' + attributes.domain,
-					attributes.secure  && '; secure'
+					(attributes.secure  && '; secure') || '' // use '' for falsy values, because [NaN, false, 0].join('') === 'NaNfalse0;
```

Comment:
Expires OK:

```shell
> new Date() && '; expires=' + new Date().toUTCString()
"; expires=Wed, 24 Jun 2015 15:35:51 GMT"
```

Path OK:
```shell
> '/' && '; path=/'
"; path=/"
```

Domain OK:
```shell
> domain.com' && '; domain=domain.com'
"; domain=domain.com"
```

Secure `true` OK:
```shell
> true  && '; secure'
"; secure"
```

Secure `false` FAIL:
```shell
> false  && '; secure'
false
```

Should not print `false`, but ignore it.